### PR TITLE
[TEST] [E2E]  Added test for setting balance and exporting gift cards

### PR DESCRIPTION
**Pull request by karola312**
## What type of PR is this?
- [ ] 💅 Refactor
- [ ] 🌟 Feature
- [ ] 🔥 Bug Fix
- [ ] 🔩 Maintenance
- [ ] 🛠 Workflow CI/CD changes

## Related Issues or Documents

- closes #

## Have you written tests?
- [ ] Yes!
- [ ] No... here is why: _Writing tests are mandatory, please replace this text with why test are not included in this PR_

### CODE REVIEW

File: playwright/pages/dialogs/exportGiftCardsDialog.ts
Comment (wojteknowacki) [inline playwright/pages/dialogs/exportGiftCardsDialog.ts:13]: Let's wait after clicking for the dialog to be gone before moving forward, WDYT?
`await this.submitButton.waitFor({state:'invisible'});`

File: playwright/pages/giftCardsPage.ts
Comment (wojteknowacki) [inline playwright/pages/giftCardsPage.ts:9]: Please use aliases in imports

File: playwright/tests/girftCards.spec.ts
Comment (wojteknowacki) [inline playwright/tests/girftCards.spec.ts:115]: Could you move it outside of the `test` body and place it in beforeEach?

File: playwright/tests/girftCards.spec.ts
Comment (wojteknowacki) [inline playwright/tests/girftCards.spec.ts:127]: Same here

File: playwright/pages/giftCardsPage.ts
Comment (wojteknowacki) [inline playwright/pages/giftCardsPage.ts:58]: Not sure is it best solution. I think `    await this.showMoreMenuButton.click();
` 
could be separate action in test body for the sake of clarity when reading test definitions. But I leave it up to you

**Review by yellowee — APPROVED** [review]

**Review by yellowee — APPROVED** [review]

File: playwright/tests/girftCards.spec.ts
Comment (wojteknowacki) [inline playwright/tests/girftCards.spec.ts:131]: It made me think that if SALEOR_113 passes then SALEOR_114 will pass even if mail won't be sent - because SALEOR_114 will find a message from SALEOR_113. Right?

Comment (wojteknowacki) [reply playwright/tests/girftCards.spec.ts:131]: If you tested it and it works then let's not overcomplicate it :)

**Review by wojteknowacki — APPROVED** [review]

File: .github/workflows/pr-automation.yml
Comment (andrzejewsky) [inline .github/workflows/pr-automation.yml:211]: it's not cypress, i think you meant `${{ secrets.MAILPITURL }}`

File: .github/workflows/run-test-cron.yml
Comment (andrzejewsky) [inline .github/workflows/run-test-cron.yml:66]: same here

File: .github/workflows/run-test-manual.yml
Comment (andrzejewsky) [inline .github/workflows/run-test-manual.yml:73]: and here

**Review by andrzejewsky — CHANGES_REQUESTED** [review]

**Review by andrzejewsky — APPROVED** [review]

**Review by andrzejewsky — COMMENTED** [review]
what's the status?

**Comment by karola312** [comment]
Waiting for review from SRE, not able to merge before that

**Review by NyanKiyoshi — APPROVED** [review]

**Pull request merged**
